From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Thu, 6 Jul 2023 09:26:22 +0200
Subject: [PATCH] ARM: at91: microchip-ksz9477-evb: migrate to
 sama5d3_barebox_entry()

We have sama5d3_barebox_entry(), so make use of it.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm/boards/microchip-ksz9477-evb/lowlevel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boards/microchip-ksz9477-evb/lowlevel.c b/arch/arm/boards/microchip-ksz9477-evb/lowlevel.c
index bda6a94160a5..aa2161daeef0 100644
--- a/arch/arm/boards/microchip-ksz9477-evb/lowlevel.c
+++ b/arch/arm/boards/microchip-ksz9477-evb/lowlevel.c
@@ -58,5 +58,5 @@ SAMA5D3_ENTRY_FUNCTION(start_sama5d3_xplained_ung8071, r4)
 
 	fdt = __dtb_z_at91_microchip_ksz9477_evb_start + get_runtime_offset();
 
-	barebox_arm_entry(SAMA5_DDRCS, SZ_256M, fdt);
+	sama5d3_barebox_entry(r4, fdt);
 }
